application/views/questionAdministration/subquestionRow.twig

```diff
@@ -129,21 +129,18 @@
         </div>
     </td>
 
-    {% set relevanceTooltip = supportsRelevance ? gT("Click to expand") : gT("Relevance is not currently supported for columns") %}
```

Comment:
Why remove this?

Reply:
Because now the column is being hidden if relevance not supported.
If the column is gone, the tooltip is not needed

Reply:
> I don't think it's the reason

What is the reason then? 

https://github.com/LimeSurvey/LimeSurvey/pull/2810/files

Reply:
> > I don't think it's the reason
> 
> What do you think is the reason then?
> 

I write my answer before seeing yours : I don't think the reason was mine «it's awfull and uneeded … it give no information»

:)